alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/main/resources/application-default.yaml

```diff
@@ -2,4 +2,10 @@ queue:
   engineRequestQueue: ${TRANSFORM_ENGINE_REQUEST_QUEUE:org.alfresco.transform.engine.imagemagick.acs}
 transform:
   config:
-    location: classpath:imagemagick_engine_config.json
\ No newline at end of file
+    location: classpath:imagemagick_engine_config.json
+imagemagick:
+  executor:
+    path:
+      root: ${IMAGEMAGICK_ROOT:/usr/lib64/ImageMagick-7.0.7}
```

Comment:
So if I want to change the imagemagick.executor.path.root property for example I should have a IMAGEMAGICK_ROOT env property?

Reply:
Yep, you can pass is as a env var like -DIMAGEMAGICK_ROOT=... and this should override the default value